CHANGELOG.rst

```diff
@@ -370,7 +370,7 @@ Fixed
 
 - Properly handle nested input object in GraphQL:
 
-  .. code-block:: graphql
+  .. code-block:: text
```

Comment:
That's not fair

Reply:
There's no `graphql` lexer provided with Pygments. However, we can try to use https://pypi.org/project/pygments-graphql/.